config/areas/system/views.php

```diff
@@ -80,11 +81,19 @@
 				];
 			}
 
+			if ($kirby->option('panel.vue.compiler', null) === true) {
+				$security[] = [
+					'id'   => 'compiler',
```

Comment:
`Suggestion`: For clarity and unambiguity:

```suggestion
					'id'   => 'vue-compiler',
```